SPDX-presentations.md

```diff
@@ -6,8 +6,10 @@ Please add talks to the top of the list (so they are listed in reverse chronolog
 
 | Date | Title | Presenter | Event | Slides/Video |
 |--|--|--|--|--|
-| 2023-09-18 | SPDX 3.0 Mini Summit | various | Open Source Summit Europe | |
-| 2023-06-13 | An Insider's Guide to SPDX|Gary O'Neall | FOSSA Webinar | [Link](https://www.brighttalk.com/webcast/17752/584752) |
+| 2023-09-26 | Package Managers, Software Security and Functional Safety | Gary O'Neall, Maximilian Huber | PackagingCon | [Link](https://cfp.packaging-con.org/2023/talk/VGS8ZS/) |
+| 2023-09-25 | SBOMs and SPDX: Now and in the Future | Gary O'Neall | FOSSA Webinar | [Link](https://www.brighttalk.com/webcast/13983/590892) |
```

Comment:
```suggestion
| 2023-09-25 | SBOMs and SPDX: Now and in the Future | Gary O'Neall | FOSSA Webinar | [Link](https://docs.google.com/presentation/d/1luX6E2GXAsq2-17eY4Gn_CvVX_knrNp7XOnjnUZJbbc) |
```

Changed link to link to the presentation - once the video is live, we can update again